### PR TITLE
28  sass partials uswds typography

### PR DESCRIPTION
**Pull request by bixgomez**
https://github.com/bixgomez/nerdstein-design-library/tree/28--sass-partials-uswds-typography

# Scope
- [/] Documentation (rationale for using partials)
- [/] Proposed design change (converting typography and colors to conform to USWDS)

# Steps Completed
- [/] Exported new compiled design change
- [/] Reviewed documentation for changes

# Notes
- "something else" directory had been auto-generated when I attempted to create a new pattern; I have deleted this directory

### CODE REVIEW

**Comment by nerdstein** [comment]
@bixgomez can you please update the description of the PR with my `Previewed Pages` section of the pull request template? This helps me to preview proposed design changes

File: design-library/source/scss/base/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
+$base-font-family: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif;
```
Comment (nerdstein) [inline design-library/source/scss/base/_variables.scss]: Are these variables and styles based on my proposed design or the USWDS work?

Comment (bixgomez) [reply design-library/source/scss/base/_variables.scss]: "Source Sans Pro" and "Merriweather" are purely in the interest of USWDS - those are the Google fonts they specify.

**Review by nerdstein — APPROVED** [review]
Approved. thank you @bixgomez

**Pull request merged**